### DISCUSSION TITLE
Play Animation with scrolling

### DISCUSSION THREAD

**Discussion by mesmaili151** [google/model-viewer#1897] [Q&A]
Hi, i want to create a long animation with Blender for my website and i want to play that with scrolling.
in this Animation, i have images and text and i have to hide/show images and test with scrolling.
its possible with model viewer?

**Comment by elalish** [comment]
Yes, I believe so; look at #1241. Hiding and showing other text and images will be done through regular DOM, unrelated to model-viewer, though they can be child elements of model-viewer if desired.
